src/App.tsx

```diff
@@ -5,7 +5,15 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (movie: Movie): void => {
+    const result = !movies.find(mov => mov.imdbId === movie.imdbId);
```

Comment:
```suggestion
    const existingMovie = !movies.find(mov => mov.imdbId === movie.imdbId);
```
стараємося називати змінні більш змістовно)